hazelcast/src/main/java/com/hazelcast/config/MultiMapConfig.java

```diff
@@ -67,28 +71,16 @@ public MultiMapConfig(String name) {
         setName(name);
     }
 
-    public MultiMapConfig(MultiMapConfig defConfig) {
-        this.name = defConfig.getName();
-        this.valueCollectionType = defConfig.valueCollectionType;
-        this.binary = defConfig.binary;
-        this.backupCount = defConfig.backupCount;
-        this.asyncBackupCount = defConfig.asyncBackupCount;
-        this.statisticsEnabled = defConfig.statisticsEnabled;
-        this.listenerConfigs = new ArrayList<EntryListenerConfig>(defConfig.getEntryListenerConfigs());
-        this.quorumName = defConfig.quorumName;
-    }
-
-    /**
-     * Gets immutable version of this configuration.
-     *
-     * @return Immutable version of this configuration
-     * @deprecated this method will be removed in 4.0; it is meant for internal usage only
-     */
-    public MultiMapConfigReadOnly getAsReadOnly() {
-        if (readOnly == null) {
-            readOnly = new MultiMapConfigReadOnly(this);
-        }
-        return readOnly;
+    public MultiMapConfig(MultiMapConfig config) {
+        this.name = config.getName();
+        this.valueCollectionType = config.valueCollectionType;
+        this.listenerConfigs = new ArrayList<EntryListenerConfig>(config.getEntryListenerConfigs());
```

Comment:
seems this line can be like this: `listenerConfigs.addAll(config.getEntryListenerConfigs())` since `listenerConfigs` already assigned to a list above.